rebar.config.erl

```diff
@@ -102,11 +102,24 @@ common_compile_opts() ->
     ].
 
 prod_compile_opts() ->
+    ExtraEncrypt =
+        case os:getenv("EMQX_COMPILE_SECRET_FILE") of
+            false ->
+                [];
+            SecretFile ->
+                io:format("debug_info encryption enabled !~n"),
```

Comment:
delete this print

Reply:
hum.. I think it is good to make it visible or?

Reply:
or move the print to where the file is read so it's not repeatedly printed for each profille.

Reply:
I will move it to do/2. rebar spawn multiple processes when do relx, read file will be called multiple times as well.

Reply:
or if we read the secret in `profiles/0`, and pass the secret down to this function, it will be logged only once.